lib/parser.js

```diff
@@ -30,10 +30,10 @@ module.exports = {
 /**
  * The complete list of parse configuration options used to parse the given data.
  * @typedef {Object} ParserOptions
- * @property {String} path - Path to the AsyncAPI document. It will be used to resolve relative references. Defaults to current working dir.
- * @property {Object} parse - Options object to pass to {@link https://apidevtools.org/json-schema-ref-parser/docs/options.html|json-schema-ref-parser}.
- * @property {Object} resolve - Options object to pass to {@link https://apidevtools.org/json-schema-ref-parser/docs/options.html|json-schema-ref-parser}.
- * @property {Boolean} applyTraits - Whether to resolve and apply traits or not. Defaults to true.
+ * @property {String=} path - Path to the AsyncAPI document. It will be used to resolve relative references. Defaults to current working dir.
+ * @property {Object=} parse - Options object to pass to {@link https://apidevtools.org/json-schema-ref-parser/docs/options.html|json-schema-ref-parser}.
+ * @property {Object=} resolve - Options object to pass to {@link https://apidevtools.org/json-schema-ref-parser/docs/options.html|json-schema-ref-parser}.
+ * @property {Boolean=} applyTraits - Whether to resolve and apply traits or not. Defaults to true.
```

Comment:
This is actually all you need to commit, the rest will be auto-generated by the CI system. This way we are sure no collisions occur. Please revert the changes in `types.d.ts` and `API.md`

Reply:
@jonaslagoni thanks for the quick review. :) it's ready.